label_studio/data_export/api.py

```diff
@@ -197,14 +197,13 @@ def get(self, request, *args, **kwargs):
             ).data
         logger.debug('Prepare export files')
 
-        export_stream, content_type, filename = DataExport.generate_export_file(
+        export_file, content_type, filename = DataExport.generate_export_file(
             project, tasks, export_type, download_resources, request.GET
         )
 
-        response = HttpResponse(File(export_stream), content_type=content_type)
-        response['Content-Disposition'] = 'attachment; filename="%s"' % filename
-        response['filename'] = filename
-        return response
+        r = FileResponse(export_file, as_attachment=True, content_type=content_type, filename=filename)
+        r['filename'] = filename
```

Comment:
response['Content-Disposition'] = 'attachment; filename="%s"' % filename

it's important to have this header for export, as I remember SDK depends on it and some customers too

Reply:
FileResponse already does this! 
![image](https://github.com/HumanSignal/label-studio/assets/3943358/598a5e84-9c2e-471d-b525-637838ebb6ea)